src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileName, true))) {
+            for (byte symbol : data) {
+                writer.write(symbol);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file");
```

Comment:
add the cause exception please
```suggestion
            throw new RuntimeException("Can't write data to file", e);
```